target/linux/realtek/dts/rtl9302_zyxel_xgs1250-12-a1.dts

```diff
@@ -1,271 +1,14 @@
 // SPDX-License-Identifier: GPL-2.0-or-later
 /dts-v1/;
```

Comment:
should be `dts-a1`

Reply:
How did you come up with the idea that the identifier  `/dts-v1/` is wrong?

According to the devicetree specification:

> The /dts-v1/; shall be present to identify the file as a version 1 DTS (dts files without this tag will be treated by dtc as being in the obsolete version 0, which uses a different format for integers in addition to other small but incompatible changes).

Reply:
Should do early morning reviews. 
You are right.

Sorry for the noice.